fpm.toml

```
@@ -0,0 +1,16 @@
+name = "stdlib"
+
+[dev-dependencies]
+test-drive.git = "https://github.com/fortran-lang/test-drive.git"
+
+[build]
+auto-examples = false
+auto-tests = true
+
+[preprocess.cpp]
```

Should it be 
```suggestion
[preprocess.fypp]
```
?
As far as I remember, CPP is not used within stdlib.

---

Yes, that is not used. But, we have not enabled ``fypp`` inside ``fpm`` as of now. So, it cannot identity it as a valid preprocessor. Considering ``fypp-gfortran.py`` as a temporary solution, I think when we have ``fypp`` integrated inside fpm then I think we should rename it as fypp instead of cpp.